### DISCUSSION TITLE
Repositories not showing

### DISCUSSION THREAD

**Discussion by Jackyyyyyyyyyyy** [community/community#30139] [Other Feature Feedback, Questions, & Ideas]
Hi, 
I havent been on GitHub for months and now when i go under my repositories i dont see any of them but it actually show 10 repo 

![image](https://user-images.githubusercontent.com/64966385/185696840-510206f2-cb3a-48ce-86c1-24207e1228ca.png)

Thanks for your help

**Comment by Zoltus** [comment]
> Hi, I havent been on GitHub for months and now when i go under my repositories i dont see any of them but it actually show 10 repo
> 
> 
> Thanks for your help

Click type and press all and also languages and all, I hope that works. if it does mark as answered;:3
![chrome_2022-08-20_00-45-59](https://user-images.githubusercontent.com/23402980/185711469-15a72c15-9ca3-4714-a232-24f8cd7e3613.png)

**Reply by Jackyyyyyyyyyyy** [reply]
Yeah already did that but no repositories are showing

**Comment by jfullstackdev** [comment]
there is none on my end, it's just maybe some caching issue

![image](https://user-images.githubusercontent.com/47092464/185750447-63ddffd4-de0a-4c56-80b9-c9e6c32ffc27.png)

**Reply by Zoltus** [reply]
if his repositys are private u cant see them

**Reply by jfullstackdev** [reply]
yes, I'm aware of that, what I'm referring to are those similar posts with incorrect repo count but I don't know whether he is referring to his private repos

**Comment by antemasqued** [comment]
@Jackyyyyyyyyyyy you should try to open up your repositories page in an incognito session (be sure to login while in incognito). I'm not showing that you have any public repos, so this is likely a caching error.

```
(base)  ──── ─ curl https://api.github.com/users/Jackyyyyyyyyyyy                                                                                                                                                                                           ~
{
  "login": "Jackyyyyyyyyyyy",
  "id": 64966385,
  "node_id": "MDQ6VXNlcjY0OTY2Mzg1",
  "avatar_url": "https://avatars.githubusercontent.com/u/64966385?v=4",
  "gravatar_id": "",
  "url": "https://api.github.com/users/Jackyyyyyyyyyyy",
  "html_url": "https://github.com/Jackyyyyyyyyyyy",
  "followers_url": "https://api.github.com/users/Jackyyyyyyyyyyy/followers",
  "following_url": "https://api.github.com/users/Jackyyyyyyyyyyy/following{/other_user}",
  "gists_url": "https://api.github.com/users/Jackyyyyyyyyyyy/gists{/gist_id}",
  "starred_url": "https://api.github.com/users/Jackyyyyyyyyyyy/starred{/owner}{/repo}",
  "subscriptions_url": "https://api.github.com/users/Jackyyyyyyyyyyy/subscriptions",
  "organizations_url": "https://api.github.com/users/Jackyyyyyyyyyyy/orgs",
  "repos_url": "https://api.github.com/users/Jackyyyyyyyyyyy/repos",
  "events_url": "https://api.github.com/users/Jackyyyyyyyyyyy/events{/privacy}",
  "received_events_url": "https://api.github.com/users/Jackyyyyyyyyyyy/received_events",
  "type": "User",
  "site_admin": false,
  "name": null,
  "company": null,
  "blog": "",
  "location": null,
  "email": null,
  "hireable": null,
  "bio": null,
  "twitter_username": null,
  "public_repos": 0,
  "public_gists": 0,
  "followers": 0,
  "following": 0,
  "created_at": "2020-05-07T12:45:46Z",
  "updated_at": "2022-09-02T16:50:23Z"
}
(base)  ──── ─ curl https://api.github.com/users/Jackyyyyyyyyyyy/repos                                                                                                                                                                                     ~
[

]```

**Comment by jfullstackdev** [comment]
try logging in in Incognito mode
